handlers/response/writer.go

```diff
@@ -0,0 +1,36 @@
+package response
```

Comment:
Consider if this package should be under the 'handlers' directory. Not sure if there are any conventions here, but currently there are only actual handlers in there. Should it only be handlers in that directory or anything handler related?